src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Input/KeyboardNavigation.cs

```diff
@@ -903,7 +903,7 @@ private void ShowFocusVisual(DependencyObject element)
                     // then we load the default FocusVisualStyle from ResourceDictionary.
                     if (fvs == FrameworkElement.DefaultFocusVisualStyle)
                     {
-                        fvs = SystemResources.FindResourceInternal(SystemParameters.FocusVisualStyleKey) as Style;
+                        fvs = FrameworkElement.FindResourceInternal(fe, null /* fce */, SystemParameters.FocusVisualStyleKey) as Style;
```

Comment:
Use named argument instead of comment (This is probably inspired from other places in the code base that uses comments but I think we should still use named argument in new code)
```suggestion
                        fvs = FrameworkElement.FindResourceInternal(fe, fce: null, SystemParameters.FocusVisualStyleKey) as Style;
```